core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -141,6 +141,16 @@ package object config {
         "Ensure that memory overhead is a double greater than 0")
       .createWithDefault(0.1)
 
+  private[spark] val STRUCTURED_LOGGING_ENABLED =
+    ConfigBuilder("spark.log.structuredLogging.enabled")
```

Comment:
this config needs to be documented on configuration.md

Reply:
Thanks, I just created https://issues.apache.org/jira/browse/SPARK-47671 for this.